### ISSUE TITLE
Hardcode Token logo.

### ISSUE THREAD

**Issue by Mantisrei** [AlphaWallet/alpha-wallet-android#2982]
Hi @JamesSmartCell , @seabornlee I added a custom token,  but there's no logo since we haven't reached a certain number of users.  Is ther a way I could hardcode the icon.

**Comment by JamesSmartCell** [comment]
Yes, submit a PR to this repo https://github.com/AlphaWallet/iconassets

Follow the format there (lowercase address, use logo.PNG) and say what your token is. This will only work on AW, forks and wallets which use this repo for icon assets.

**Comment by JamesSmartCell** [comment]
Alternatively for testing you can add your icon to the resources ( via AndroidStudio) and then add a pickup in TokenIcon.java which detects your chain and address and uses that resource.

**Comment by Mantisrei** [comment]
Will try that asap.  Coming
